### DISCUSSION TITLE
Should I use class button inside of an Block or should I create another class elemen__button and new style for It.

### DISCUSSION THREAD

**Discussion by abhinavnepal7** [getbem/getbem.github.io#299] [Q&A]
Just red the title.

**Comment by dotnetCarpenter** [comment]
it depends. https://en.bem.info/methodology/css/#mixes

Some say that you should create a new block if it does not have any external size, e.i. margin, width, height etc. and then you can optionally add style to it within a block as a block__element.

I would probably style it as part of a bigger block and the first time I copy styles to another block, I can take the overlap CSS and have a _base_ style for a new block, while my two existing blocks can add a few more styles that are different because of the context.
